### PR TITLE
Update actionable-messages-faq.md

### PR DESCRIPTION
**Pull request by avijityadav**
We support updating any details instead of just email address. Made minor updates.

### CODE REVIEW

File: docs/actionable-messages/actionable-messages-faq.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ To check who are your organization IT admins, follow the following steps :
 1. In the response you can search for **Exchange Administrator** and **Global Administrator**.
 1. In some tenants **Exchange Administrator** might not be present. If the **Exchange Administrator** role is not listed then you can use the `id` from the **Global Administrator** role to fetch the list of tenant admins.
 
-### How can I update sender email address for an approved registered provider?
+### How can I update any details for an approved registered provider?
 
-To update the sender list for you Actionable Message provider, you can reach out to [onboardoam@microsoft.com](mailto:onboardoam@microsoft.com) with updated details. Our team will acknowledge and update it in backend. Please note that any update will take 2 weeks after the team acknowledges your update.
+To update any details for an approved Actionable Message provider, you can reach out to [onboardoam@microsoft.com](mailto:onboardoam@microsoft.com) with updated details. Our team will acknowledge and update it in backend. Please note that any update will take 2 weeks after the team acknowledges your update.
```
Comment (SiddharthaVinnakote) [inline docs/actionable-messages/actionable-messages-faq.md:42]: Consider reword
To update details for an approved global scoped Actionable Message provider, .......

File: docs/actionable-messages/actionable-messages-faq.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ To check who are your organization IT admins, follow the following steps :
 1. In the response you can search for **Exchange Administrator** and **Global Administrator**.
 1. In some tenants **Exchange Administrator** might not be present. If the **Exchange Administrator** role is not listed then you can use the `id` from the **Global Administrator** role to fetch the list of tenant admins.
 
-### How can I update sender email address for an approved registered provider?
+### How can I update any details for an approved registered provider?
```
Comment (SiddharthaVinnakote) [inline docs/actionable-messages/actionable-messages-faq.md:40]: consider reword
How can I update details for an approved global scoped Actionable message registration

Comment (avijityadav) [reply docs/actionable-messages/actionable-messages-faq.md]: Done

File: docs/actionable-messages/actionable-messages-faq.md (hunk 1 above)
Comment (avijityadav) [reply docs/actionable-messages/actionable-messages-faq.md]: Done

**Review by SiddharthaVinnakote — APPROVED** [review]

**Pull request merged**